go/pkg/bertymessenger/api.go

```diff
@@ -297,21 +297,28 @@ func (svc *service) SendContactRequest(ctx context.Context, req *SendContactRequ
 		return nil, errcode.TODO.Wrap(err)
 	}
 
-	go svc.replicateContactGroupOnAllServers(req.BertyID.AccountPK)
+	go svc.autoReplicateContactGroupOnAllServers(req.BertyID.AccountPK)
```

Comment:
Didn't see the goroutine here before, isn't this introducing races?

Reply:
Should be fine, what race condition are you expecting there?

I just want to avoid failing requests on this call due to errors on this optional feature (and return a "success" value asap)

Reply:
I dont expect something, I was asking to know if it was taken into consideration
There is an implicit (we should probably make it explicit) "contract" that only one thread will modify the `svc` by using a mutex in every handler and api call, this might escape the mutex lock of the call

Reply:
But I think since it's already there it's outside the scope of this PR so this was just to trigger the discussion, we should open an issue propably